### DISCUSSION TITLE
STEP File

### DISCUSSION THREAD

**Discussion by mrelhussein** [gismo/gismo#525] [Q&A] [answered]
How can I read STEP files in kirchoff love example ?
Thank you in advance
![Capture](https://user-images.githubusercontent.com/102301423/159921209-f2772cb2-c51a-473b-bbc9-80fdda6f5059.PNG)

**Comment by hverhelst** [comment, marked as answer]
Dear @mrelhussein ,

I would say that if you want to use shells using external files, the `kirchhoff-Love_example.cpp` is not the way to go. Instead, you can make use of (for example) the file `gsStructuralAnalysis/gsThinShell_Static_XML.cpp`, which takes the boundary value problem and the geometry from XML. If G+Smo succeeds in reading your STEP file into a `gsGeometry` or a `gsMultiPatch`, then this example should be a good starting point. 
Let me know how it goes!
